examples/docker/modin-omnisci/Dockerfile

```diff
@@ -32,25 +39,37 @@ RUN adduser --disabled-password \
     --home $HOME \
     $USER
 
-ENV CONDA_DIR ${HOME}/miniconda
-
-SHELL ["/bin/bash", "--login", "-c"]
+# Conda settings
+ENV CONDA_DIR=${HOME}/miniconda
+ENV CONDA_ENV_NAME=modin-omnisci
+ENV PATH="${CONDA_DIR}/bin:${PATH}"
 
 RUN wget -nv https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda3.sh && \
     bash /tmp/miniconda3.sh -b -p "${CONDA_DIR}" -f -u && \
     "${CONDA_DIR}/bin/conda" init bash && \
-    rm -f /tmp/miniconda3.sh && \
-    echo ". '${CONDA_DIR}/etc/profile.d/conda.sh'" >> "${HOME}/.profile"
+    rm -f /tmp/miniconda3.sh
 
 RUN conda update -n base -c defaults conda -y && \
-    conda config --set channel_priority strict && \
-    conda create -n modin --yes --no-default-packages && \
-    conda activate modin && \
-    conda install -c conda-forge modin-omnisci scikit-learn scikit-learn-intelex xgboost && \
+    conda create -n ${CONDA_ENV_NAME} --yes -c conda-forge modin-omnisci scikit-learn scikit-learn-intelex xgboost && \
     conda clean --all --yes
 
-COPY plasticc-omnisci.py "${HOME}/plasticc-omnisci.py"
+# Activate ${CONDA_ENV_NAME} for interative shells
+RUN echo "source ${CONDA_DIR}/bin/activate ${CONDA_ENV_NAME}" >> "${HOME}/.bashrc"
+# Activate ${CONDA_ENV_NAME} for non-interative shells
+RUN sed -e 's,\(^[[:space:]]\+[*]) return;;$\),# \1,' -i "${HOME}/.bashrc"
```

Comment:
Please add comments on what this line does - to make the `sed` syntax easier to read.

Reply:
Ok.